server/routes/profile.js

```diff
@@ -3,12 +3,12 @@ const router = express.Router();
 const protect = require("../middleware/auth");
 const {
   profileCreatePost,
-  profileUpdatePost,
+  profileUpdatePatch,
   profileGet,
   profileGetAll,
 } = require("../controllers/profile");
-router.route("/create").post(protect, profileCreatePost);
-router.route("/update").post(protect, profileUpdatePost);
+router.route("/").post(protect, profileCreatePost);
+router.route("/").patch(protect, profileUpdatePatch);
```

Comment:
Add the `id` of the resource you intend to update in the request params `'/:id'`